### PR TITLE
config updates for clearity and better defaults

### PR DESCRIPTION
**Pull request by yairchn**
# Modulus Pull Request

## Description
two config updates that I think are useful:
1. the name of the seed parameter in generation is not clearly linked to what is does, I changes the text next to it 
2. the dump parameter training was set to a value unsuitable for a 4h job, I reduced it and updated the text

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [x] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: examples/generative/corrdiff/conf/config_generate.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ reg_ckpt_filename: "./checkpoints/regression.mdlus"
 image_outdir: "image_outdir"
   # Where to save the output images
 seeds: "0-63"
-  # Random seeds used for generation
+  # Number of samples on the ensemble dimension
```
Comment (mnabian) [inline examples/generative/corrdiff/conf/config_generate.yaml:30]: This is technically not the number of samples, but the random seeds that are used for each ensemble

File: examples/generative/corrdiff/conf/config_train_base.yaml [hunk 2]
```diff
@@ -64,8 +64,8 @@ checkpoint_dir: checkpoints
   # Where to save the checkpoints
 tick: 1
   # How often to print progress
-dump: 500
-  # How often to dump state
+dump: 50
+  # How often to write a ckpt (diff needs more than reg)
```
Comment (mnabian) [inline examples/generative/corrdiff/conf/config_train_base.yaml:68]: "diff needs more than reg" I'm not sure if I understand this

File: examples/generative/corrdiff/conf/config_generate.yaml (hunk 1 above)
Comment (yairchn) [reply examples/generative/corrdiff/conf/config_generate.yaml:30]: @mnabian 
if the user wants to run a generation with 100 members, that parameter controls that number right? there is nowhere else one can control the number of members.

Comment (mnabian) [reply examples/generative/corrdiff/conf/config_generate.yaml:30]: That is correct

**Pull request closed without merging**